rotkehlchen/db/dbhandler.py

```diff
@@ -1537,7 +1537,13 @@ def write_tuples(
             write_cursor.executemany(query, tuples)
             if relevant_address is not None:
                 # chain_id should always be the same so perform lookup out of the loop
-                blockchain = ChainID(tuples[0][1]).to_blockchain().value
+                try:
+                    blockchain = ChainID(tuples[0][1]).to_blockchain().value
```

Comment:
This is hacky here. Plus we may need to have similar logic for other chains (maybe arbitrum?) and then this logic will break.

Any such transformations should happen at the caller at dbevmtx.py.

Reply:
The problem I was trying to solve with these actually went away once I took out the relevant_address from optimismtx's call to write_tuples, so I simply restored to the original